inst/codec_data/property_code_enforcements/property_code_enforcements.R

```diff
@@ -0,0 +1,80 @@
+if (tryCatch(read.dcf("DESCRIPTION")[1, "Package"] == "codec", finally = FALSE)) {
+  devtools::load_all()
+} else {
+  library(codec)
+}
+message("Using CoDEC, version ", packageVersion("codec"))
+library(dplyr)
+library(sf)
+library(dpkg)
+library(addr)
+options(arrow.unsafe_metadata = TRUE)
+
+code_enforcement_url <- "https://data.cincinnati-oh.gov/api/views/cncm-znd6/rows.csv?accessType=DOWNLOAD"
+
+raw_data <-
```

Comment:
I decided to use the lat/lon that comes with the raw data rather than matching to parcel then joining the parcel lat/lon since it's just being aggregated to tract. Then I use the number of parcels per tract by summing cagis_parcels for each tract. Is violations **per parcel** still the best way to summarize ?